sass/_ExplanationModal.scss

```diff
@@ -58,6 +58,10 @@
       align-items: flex-start;
       flex-grow: 1;
 
+      &.disabled {
```

Comment:
Is the disabled class a bit too generic? We used `coveo-hidden` a lot in the past

Reply:
Good point, I'll use `coveo-hidden`.